src/main/java/com/guysfromusa/carsgame/v1/CarResource.java

```diff
@@ -0,0 +1,47 @@
+package com.guysfromusa.carsgame.v1;
+
+
+import com.guysfromusa.carsgame.entities.CarEntity;
+import com.guysfromusa.carsgame.entities.enums.CarType;
+import com.guysfromusa.carsgame.services.CarService;
+import com.guysfromusa.carsgame.v1.converters.CarConverter;
+import com.guysfromusa.carsgame.v1.model.Car;
+import org.springframework.web.bind.annotation.*;
+
+import javax.inject.Inject;
+
+import java.util.List;
+
+import static java.util.Collections.singletonList;
+import static org.springframework.http.MediaType.APPLICATION_JSON_UTF8_VALUE;
+
+@RestController
+@RequestMapping(value= "/v1/cars", produces =  APPLICATION_JSON_UTF8_VALUE)
+public class CarResource {
+
+    private final CarService carService;
+
+    @Inject
+    public CarResource(CarService carService) {
+        this.carService = carService;
```

Comment:
notNull(carService) to fail fast during spring initialization (once bean was not injected -> forgotten @Component or sth)